css/style.css

```diff
@@ -1,11 +1,163 @@
 /* Simple CSS Reset */
 * {
-  margin: 0;
-  padding: 0;
+  	margin: 0;
+  	padding: 0;
+}
+body {
+	color: #f0efef;
+  font-size: 26px;
+  line-height: 1.4;
+  color: #606161;
+  font-family: "Open Sans", Helvetica, Arial, sans-serif;
+  font-weight: 300;
+}
+a {
+	text-decoration: none;
 }
-
 header {
```

Comment:
You can delete this entire selector / ruleset. All these properties are defined below.